pkg/settings/setting_test.go

```diff
@@ -0,0 +1,35 @@
+package settings
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestIsRelease(t *testing.T) {
+	inputs := map[string]bool{
+		"dev":         false,
+		"master-head": false,
+		"master":      false,
+		"v2.5.2":      true,
+		"v2":          true,
+		"v2.0":        true,
+		"v2.x":        true,
+		"v2.5-head":   false,
+		"2.5":         false,
+		"2.5-head":    false,
+	}
```

Comment:
Creating a `a := assert.New(t)` here will allow you to stop passing in T down below into the test.

Reply:
Did not know this. Thanks for the suggestion.